src/components/markdownRenderer/markdown-renderer.module.css

```diff
@@ -0,0 +1,101 @@
+.markdown h1 {
+    font-size: 48px;
+    color: black;
+    background: linear-gradient(
+        180deg,
+        rgba(255, 255, 255, 0) 55%,
+        #7cf952 55%
+    );
+    text-decoration: underline;
+    text-decoration-thickness: 19px;
+    text-decoration-color: #7cf952;
+    display: inline;
+}
+
+.markdown h2 {
+    font-size: 48px;
+    color: black;
+    display: inline;
+}
+
+.markdown h3 {
+    font-size: 32px;
+    color: black;
+    text-decoration: underline;
+    text-decoration-thickness: 2px;
+    text-underline-offset: 5px;
+    text-decoration-color: #7cf952;
+}
+
+.markdown h4 {
+    font-size: 32px;
+    color: black;
+}
+
+.markdown h5 {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 12px;
+    color: black;
+}
+
+.markdown p {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 12px;
+    color: gray;
+}
+
+.markdown pre {
+    font: inherit;
+}
+
+.markdown code {
+    white-space: pre;
+    text-decoration: none !important;
+    text-decoration-color: white;
+}
```

Comment:
do we need/use the `code` formatting?

Reply:
yea so basically 'code' isn't actually the monospace stuff you would expect, it's what I use to be able to process indentation via $

Essentially, white space is usually reduced to a single space by the DOM (there was a good reason but don't remember why), so I used the 'code' styling to force whitespace